.taskcluster.yml

```diff
@@ -39,7 +39,7 @@ tasks:
       deadline: {$fromNow: '1 hour'}
       payload:
         maxRunTime: 3600
-        image: python:3.7
+        image: python:3.9
```

Comment:
Is the upgrade to 3.9 necessary for this PR?

Reply:
Yes, due to the `:=` operator used in this line https://github.com/mozilla/task-boot/pull/232/commits/8e0e46f600639b8473650d0d9fb4aa28fa3b9958#diff-78ecac4bb1b6cbde6b52231869a11e231512fb46a02ad5b6bb4315a956dcb202R34

Reply:
You can avoid using it so we don't have to upgrade now. In that case it doesn't improve readability at all, so we might as well use the good old `=`.